release-1.6/release-notes-draft.md

```diff
@@ -135,7 +135,7 @@ Create subcommands
 ### Features
 - **[alpha]** New Bootstrap Token authentication and management method. Works well with kubeadm. kubeadm now supports managing tokens, including time based expiration, after the cluster is launched.  See [kubeadm reference docs](https://kubernetes.io/docs/admin/kubeadm/#manage-tokens) for details.
 - **[alpha]** Adds a new cloud-controller-manager binary that may be used for testing the new out-of-core cloudprovider flow.
-- **[beta]** Introduces an API for the Kubelet to request TLS certificates from the API server.
+- **[beta]** Introduces an API for the clients to request TLS certificates from the API server. See the [tutorial](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster).
```

Comment:
Nit: "the clients" sounds a little weird here. How about just "an API for clients to request..." Otherwise LGTM.